paper/forestecology.Rmd

```diff
@@ -367,8 +366,7 @@ plot3 <- autoplot(comp_bayes_lm_scbi, type = "competition", sp_to_plot = sp_to_p
 (plot1 | plot2) / plot3
 ```
 
-TODO: Discuss meaning of results.
-
+These plots give the posterior distributions of parameters from equation 1 (DAVE NOTE:SOrry couldn't figure out how to make equation references work). For many pacakge users they will be of interest because they give insight into the species-specific competitive interactions. Setting `type = "intercepts"` gives posterior distributions for $\beta_{0,j}$ and `type = "dbh_slopes"` for $\beta_{dbh,j}$. These give species specific growth independent of competition. The values of more interest are plotted with `type = 'competition'` which gives the posterior distribution for $\lambda_{j,k}$ species-specific competition coefficients (i.e., the $\lambda$-matrix). Negative values indicate a competitor species which slows the growth of a focal species. Here, for example, we see that tulip trees (`litu`) have a strong negative effect on the growth of conspecifics but relatively little effect on neighbors of the other two species.
```

Comment:
Equation ref fixed

Reply:
it took me a surprising amount of time to figure out too. here is what I used: https://gist.github.com/dewittpe/ee42f46b23b9de6b1338